packages/serverless/src/apollo/app/transaction.ts

```diff
@@ -31,20 +32,42 @@ extend type Query {
 
 extend type Mutation {
   enqueueTransaction(databaseName: String!, transactionType: TransactionType!): Boolean
```

Comment:
It return boolean but in enqueueTransactionDomain return ObjectId

Reply:
done